Sources/NestKit/Assets/AssetRegistryClientBuilder.swift

```diff
@@ -3,21 +3,21 @@ import Logging
 
 public struct AssetRegistryClientBuilder: Sendable {
     private let httpClient: any HTTPClient
-    private let authToken: String?
+    private let serverConfigs: GitHubServerConfigs?
```

Comment:
Could you avoid  from passing a type which contains `GitHub`.
This layer should not consider an actual service.